src/dawn/Optimizer/PassSetCaches.cpp

```diff
@@ -182,6 +182,7 @@ bool PassSetCaches::run(StencilInstantiation* instantiation) {
              !field.Extent.isHorizontalPointwise()) {
 
             Cache& cache = multiStagePtr->setCache(Cache::IJ, Cache::local, field.AccessID);
+            instantiation->getCachedVariableSet().emplace(field.AccessID);
```

Comment:
`insertCacheVariable` ? 
Aside comment, this has not to do with BC, was that a bug we had?

Reply:
This was not needed before BC's